pulp_container/constants.py

```diff
@@ -43,3 +55,16 @@
 SIGNATURE_PAYLOAD_MAX_SIZE = 4 * MEGABYTE
 
 SIGNATURE_API_EXTENSION_VERSION = 2
+
+
+ALLOWED_ARTIFACT_TYPES = [MEDIA_TYPE.CONFIG_BLOB_OCI]
+ALLOWED_BLOB_CONTENT_TYPES = OCI_BLOB_MEDIA_TYPE
+
+
+def register_well_known_types(artifact_config_type, artifact_layer_types):
+    if artifact_config_type not in ALLOWED_ARTIFACT_TYPES:
+        ALLOWED_ARTIFACT_TYPES.append(artifact_config_type)
+
+    for layer_type in artifact_layer_types:
+        if layer_type not in ALLOWED_BLOB_CONTENT_TYPES:
+            ALLOWED_BLOB_CONTENT_TYPES.append(layer_type)
```

Comment:
In this function we loose all the information what blob type is associated with what config type. And i do not see it being used anywhere else. So this maybe important piece of information is lost. Still I'm unsure what we loose if we just started to accept every possible content type for blobs. Would that not just be absolutely future proof?